api/test/apitests.ts

```diff
@@ -136,17 +190,54 @@ describe('testing api', function() {
         .put('/api/CurrentKeg/1')
         .auth(process.env.BasicAuthUsername, process.env.BasicAuthPassword)
         .send({KegId: 6, KegSize: 17000})
-        .end(function(err, res){
+        .end((err: any, res: ChaiHttp.Response) => {
             res.should.have.status(401);
             done();
         })
     });
 
+    it('should require admin bearer token authentication on /api/CurrentKeg/<id> PUT', function(done) {
+        chai.request(server)
+        .put('/api/CurrentKeg/1')
+        .set('Authorization', 'Bearer ' + nonAdminBearerToken)
+        .send({KegId: 6, KegSize: 17000})
+        .end((err: any, res: ChaiHttp.Response) => {
+            res.should.have.status(401);
+            done();
+        })
+    });
+
+    it('should make previously installed keg current /api/CurrentKeg/<id> PUT', function(done) {
```

Comment:
Do we need to reset the database with a script before running the tests that change the db state?